lib/api/kuzzle.js

```diff
@@ -226,15 +216,15 @@ function registerErrorHandlers(kuzzle) {
   });
 
   // abnormal termination signals => generate a core dump
-  Object.keys(coreDumpSignals).forEach(signal => {
+  for (const signal of ['SIGQUIT', 'SIGABRT']) {
```

Comment:
nitpicking (optional) : 
I'd rather have the list of signals in a variable, it's clearer I thought

Reply:
I tend to remove variables whenever they are used in only one place, because I find it simpler that way.
I guess this is a matter of personnal preferences.

Reply:
I guess :)